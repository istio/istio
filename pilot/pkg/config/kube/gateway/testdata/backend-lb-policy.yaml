apiVersion: gateway.networking.x-k8s.io/v1alpha1
kind: XBackendTrafficPolicy
metadata:
  name: lb-policy
  namespace: default
spec:
  targetRefs:
  - group: ""
    kind: Service
    name: echo
  retryConstraint:
    minRetryRate:
      interval: "1s"
      count: 5
    budget:
      percent: 20
      interval: "10s"

  sessionPersistence:
    sessionName: foo
    absoluteTimeout: 1h
    type: Cookie
    cookieConfig:
      lifetimeType: Permanent
