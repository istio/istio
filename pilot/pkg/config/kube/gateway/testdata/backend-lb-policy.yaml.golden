apiVersion: networking.istio.io/v1
kind: DestinationRule
metadata:
  annotations:
    internal.istio.io/parents: XBackendTrafficPolicy/default.lb-policy
  creationTimestamp: null
  name: echo-istio-autogenerated-k8s-gateway
  namespace: default
spec:
  host: echo.default.svc.domain.suffix
  trafficPolicy:
    loadBalancer: {}
---
